src/quizzes/chapter-3/3.4/containers-quiz.js

```diff
@@ -0,0 +1,29 @@
+const rawQuizdown = `
+---
+shuffleQuestions: true
+shuffleAnswers: true
+---
+
+# What are the key distinctions between containers and virtual machines in terms of resource utilization? Select all that apply
+
+- [x] Containers share the host operating system's kernel, while VMs run their own operating system kernel
+	> Good. This was the big revelation with containers that keeps them lightweight and easily scalable
```

Comment:
This is a great question but would like a note about how the shared kernel has security implications. From my understanding with VM's the machines are separated at the hardware level and which means a compromised VM does not endanger the other VMs. That is not true for containers, and compromising the kernel  of the container they can gain access to other containers.

Reply:
@cfculler

Reply:
Added this point you've made as feedback when they get the question right. To ensure apprentices really internalize this tradeoff I also added it as a deliverable talking point for a DOB review